pytensor/tensor/special.py

```diff
@@ -752,9 +752,20 @@ def factorial(n):
     return gamma(n + 1)
 
 
-__all__ = [
-    "softmax",
-    "log_softmax",
-    "poch",
-    "factorial",
-]
+def beta(a, b):
+    """
+    Beta function.
+
+    """
+    return (gamma(a) * gamma(b)) / gamma(a + b)
+
+
+def betaln(a, b):
+    """
+    Log beta function.
+
+    """
+    return gammaln(a) + gammaln(b) - gammaln(a + b)
+
+
+__all__ = ["softmax", "log_softmax", "poch", "factorial", "beta", "betaln"]
```

Comment:
If you add a comma after the last entry, the pre-commit will default to multi-line, which is better for git changes

```suggestion
__all__ = ["softmax", "log_softmax", "poch", "factorial", "beta", "betaln",]
```

Reply:
Thanks for the suggestion! I didn't know about it, and I felt weird that pre-commit had arranged this line in this way, but didn't want to overwrite the pre-commit changes by myself. 

I tried to accept the suggestion through the website UI but it apparently didn't run pre-commit automatically, so I ran in my machine and pushed it again.